README.md

```diff
@@ -1,5 +1,14 @@
-# Vera iOS SDK
-Official repo for Vera iOS SDK.
+<p align="center">
+    <img alt="Vera: A computer vision enterprise platform that transforms buildings into intelligent environments" src="./Vera.png">
+</p>
+<p align="center">
+A computer vision enterprise platform that transforms buildings into intelligent environments.
+</p>
+
+<p align="center">
+    <a href="https://developer.apple.com/swift/"><img alt="Swift 5.7" src="https://img.shields.io/badge/swift-5.7-orange.svg?style=flat"></a>
+    <a href="https://github.com/resonai/vera-ios-sdk/releases"><img alt="Vera Release" src="https://img.shields.io/github/v/release/resonai/vera-ios-sdk"></a>
+</p>
 
 ## Installation
```

Comment:
Should we mention that AR Kit is required?

Reply:
No, it is supported by default (unlike Android).